admin/resources/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -0,0 +1,338 @@
+              ]}
+              rules={{ required: errorMessages.required }}
+            />
+            <Input
+              id="cmoIdentifier"
+              label="Cmo Identifier: "
+              type="text"
+              name="cmoIdentifier"
+              rules={{ required: errorMessages.required }}
+            />
+            <Input
+              id="expiryDate"
+              label="Expiry Date: "
+              type="date"
+              name="expiryDate"
+              rules={{
+                required: errorMessages.required,
+                min: {
+                  value: new Date().toISOString().slice(0, 10),
+                  message: errorMessages.futureDate,
+                },
+              }}
+            />
+            <Input
+              id="isWoman"
+              label="Woman: "
+              type="checkbox"
+              name="isWoman"
+            />
+            <Input
+              id="hasDisability"
+              label="Has Disability: "
+              type="checkbox"
+              name="hasDisability"
+            />
+            <Input
+              id="isIndigenous"
+              label="Indigenous: "
+              type="checkbox"
+              name="isIndigenous"
+            />
+            <Input
+              id="isVisibleMinority"
+              label="Visible Minority: "
+              type="checkbox"
+              name="isVisibleMinority"
+            />
+            <Input
+              id="hasDiploma"
+              label="Has Diploma: "
+              type="checkbox"
+              name="hasDiploma"
+            />
+            <Select
+              id="languageAbility"
+              label="Language Ability: "
+              name="languageAbility"
+              options={[
+                { value: "", label: "Select a language ability..." },
+                ...enumToOptions(LanguageAbility),
+              ]}
+              rules={{ required: errorMessages.required }}
+            />
+            <MultiSelect
+              id="locationPreferences"
+              name="locationPreferences"
+              label="Location Preferences: "
+              placeholder="Select one or more location preferences..."
+              options={enumToOptions(WorkRegion)}
+              rules={{ required: errorMessages.required }}
+            />
+            <MultiSelect
+              id="acceptedOperationalRequirements"
+              name="acceptedOperationalRequirements"
+              label="Operational Requirements: "
+              placeholder="Select one or more operational requirements..."
+              options={operationalRequirementOptions}
+              rules={{ required: errorMessages.required }}
+            />
+            <MultiSelect
+              id="expectedSalary"
+              label="Expected Salary: "
+              name="expectedSalary"
+              placeholder="Select one or more expected salaries..."
+              options={enumToOptions(SalaryRange)}
```

Comment:
We might want to map salary enum to other strings for display purposes. eg 
```
{
  "50_59K": "$50,000 - $59,999",
  "60_69K": "$60,000 - $69,999",
}
```

Reply:
With different versions for en and fr. Not sure where to put this constant / helper function. Its similar to localizedConstants we had in the TC project.

Reply:
(Same goes for UpdatePoolCandidate).